bin/console

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
```

Comment:
That's cool!

Reply:
Yes, a little `bin/console` is quite nice for ad-hoc debugging and experimenting.